sdk/java/src/main/java/com/equinix/pulumi/metal/outputs/GetProjectBgpConfig.java

```diff
@@ -31,7 +29,7 @@ public final class GetProjectBgpConfig {
      * @return Password for BGP session in plaintext (not a checksum).
      * 
      */
-    private @Nullable String md5;
+    private String md5;
```

Comment:
I don't understand what this change corresponds to on the terraform side.  The [md5 attribute is still optional in the terraform provider](https://github.com/equinix/terraform-provider-equinix/blob/main/internal/resources/metal/project/resource_schema.go#L105-L108), so I would expect it to remain nullable in Pulumi.

Reply:
When using the terraform provider directly, at least, MD5 should still be nil on output if the user provides a BGP config but does not specify an MD5; that behavior was restored in 1.33.1 and should continue to exist in 1.34.0.

Reply:
I'm on board to write this off as a Pulumi thing, but my concern is that this may indicate there's still something wrong with how we're handling null MD5 in the Terraform provider.

Reply:
you right I was checking bgpConfig instead of the specific md5 field

Reply:
It sounds like this is a reflection of the issue we ran into during the TF framework provider conversion, where we couldn't define a nested schema for BGP config in the data source, so it doesn't have the same behavior as the converted resource.  If that's the case, this is expected and there's not really anything we can do about it.